src/pages/PlaylistManagementPage/PlaylistManagementPage.js

```diff
@@ -10,8 +10,9 @@ const PlaylistManagementPage = (props) => {
 
   useEffect(() => {
     Promise.all([
-      //axios.get(`${process.env.REACT_APP_API_BASE_URL}/playlist`, {withCredentials: true}),
-      axios.get(`${process.env.REACT_APP_API_BASE_URL}/me/playlist?collaborative=true&&mine=true`, {
+      // Query parameter 'collaborative' was removed due a problem in the consumed API 
+      // More explanation at https://github.com/yurigava/democratic-spotify-playlist/issues/34 
```

Comment:
I think these comments should be removed. Documentation and discussions can be left in github

Reply:
Done